src/components/ArticlesList/ArticlesList.tsx

```diff
@@ -1,20 +1,37 @@
-import {FlatList} from 'react-native';
+import {TouchableOpacity} from 'react-native';
 import React from 'react';
 
 import {ArticlesListItem} from './ArticlesListItem';
 import {StyledViewComp} from '../SimpleComponents/StyledViewComp';
-import {useAppSelector} from '../../redux/hooks';
+import {useAppDispatch, useAppSelector} from '../../redux/hooks';
+import DraggableFlatList, {
+  RenderItemParams,
+} from 'react-native-draggable-flatlist';
+import {ArticleData} from '../../types/ArticlesData';
+import {setArticlesListOrder} from '../../redux/ArticlesListSlice';
 
 const separator = () => <StyledViewComp height="1px" backgroundColor="grey" />;
```

Comment:
This is a component
-component naming should be capitalised
-should be placed in simple components folder